ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import getpass
+import os
+import subprocess
+from subprocess import PIPE
+
+from ros2pkg.api.create import create_folder
+from ros2pkg.api.create import create_template_file
+from ros2pkg.verb import VerbExtension
+
+
+class CreateVerb(VerbExtension):
+    """Create a new ROS2 package."""
+
+    def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            'package_name',
+            help='The package name')
+        parser.add_argument(
+            '--destination-directory',
+            default=os.curdir,
+            help='Directory where to create the package')
+        parser.add_argument(
+            '--build-tool',
+            default='ament_cmake',
+            choices=['cmake', 'ament_cmake'],
+            help='Which build tool to use')
+        parser.add_argument(
+            '--dependencies',
+            nargs='+',
+            default=[],
+            help='list of dependencies')
+        parser.add_argument(
+            '--maintainer-email',
+            nargs='?',
+            help='email address of the maintainer of this package'),
+        parser.add_argument(
+            '--maintainer-name',
+            nargs='?',
+            help='name of the maintainer of this package'),
+        parser.add_argument(
+            '--cpp-node-name',
+            nargs='?',
+            help='name of the empty cpp executable')
+        parser.add_argument(
+            '--cpp-library-name',
+            nargs='?',
+            help='name of the empty cpp library')
+
+    def main(self, *, args):
+
+        maintainer_name = getpass.getuser()
+        if not args.maintainer_name:
+            # try getting the name from the global git config
```

Comment:
Is the git username a good fallback? I would expect that it almost never has a "normal" firstname + lastname.

Reply:
I think it's valid fallback. At least the information inside the package.xml corresponds to github commits (if any).

Do you have any ideas?

Reply:
The current username of the system could be one. But maybe also potentially not useful...